### PR TITLE
changed js to ts and added loader to movie card

### PR DESCRIPTION
**Pull request by NataliaHuberna**

### CODE REVIEW

File: src/styles/movie.css [hunk 1]
```diff
@@ -54,4 +54,82 @@
...
+    display: none;
+}
+
+.lds-ellipsis {
+    display: inline-block;
+    position: relative;
+    width: 80px;
+    height: 80px;
+    top: 50%;
+    left: 50%;
```
Comment (duhenets) [inline src/styles/movie.css:83]: add -> transform: translate(-50%, -50%);

File: src/styles/movie.css [hunk 2]
```diff
@@ -54,4 +54,82 @@
...
+
+.loader-full-screen {
+    width: 100%;
+    height: 100vh;
+    background: rgba(0,0,0,0.9);
+    position:fixed;
+    top:0;
+    left:0;
+    align-items: center;
+    justify-content: center;
```
Comment (duhenets) [inline src/styles/movie.css]: Why are these properties there and "display: flex;" is not?

File: src/styles/movie.css [hunk 3]
```diff
@@ -54,4 +54,82 @@
...
     line-height: 1;
     letter-spacing: 0.1em;
+}
+#genres {
+    min-height: 40px;
+}
+
+.loader-full-screen {
+    width: 100%;
+    height: 100vh;
```
Comment (duhenets) [inline src/styles/movie.css]: change to %

File: src/styles/movie.css [hunk 4]
```diff
@@ -54,4 +54,82 @@
...
+    background: rgba(0,0,0,0.9);
```
Comment (duhenets) [inline src/styles/movie.css]: to css variable

File: src/scripts/movie/movie.js [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
+import { URL_GENRES, URL_IMG, CLASS_HIDDEN } from '../constants';
 import {
     clearPagesFromLocalStorage,
     formatRate,
     formatRuntime,
     loadPageFromLocalStorage,
 } from '../movies/helpers';
 
+export const removePageLoader = () => {
+  domElements.loaderFullScreen.classList.toggle(CLASS_HIDDEN);
```
Comment (duhenets) [inline src/scripts/movie/movie.js]: fix indent

File: src/scripts/movies/global-var.ts [hunk 6]
```diff
@@ -11,7 +12,7 @@ export const domElements = {
     releaseDateLast: document.querySelector('#release-date-last'),
 };
 
-export const globalVar = {
+export const globalVar: TglobalVar = {
```
Comment (duhenets) [inline src/scripts/movies/global-var.ts]: camelCase

**Review by duhenets — CHANGES_REQUESTED** [review]
fix comments

**Review by duhenets — APPROVED** [review]

**Review by ratroian — APPROVED** [review]

**Pull request merged**